rpm/BUILD.bazel

```diff
@@ -159,6 +159,76 @@ rpmtree(
+        "@alternatives-0__1.24-1.el9.x86_64//rpm",
+        "@basesystem-0__11-13.el9.x86_64//rpm",
+        "@bash-0__5.1.8-6.el9.x86_64//rpm",
+        "@bzip2-libs-0__1.0.8-8.el9.x86_64//rpm",
+        "@ca-certificates-0__2023.2.60_v7.0.306-90.1.el9.x86_64//rpm",
+        "@centos-gpg-keys-0__9.0-23.el9.x86_64//rpm",
+        "@centos-stream-release-0__9.0-23.el9.x86_64//rpm",
+        "@centos-stream-repos-0__9.0-23.el9.x86_64//rpm",
+        "@coreutils-single-0__8.32-34.el9.x86_64//rpm",
+        "@crypto-policies-0__20230731-1.git94f0e2c.el9.x86_64//rpm",
+        "@curl-minimal-0__7.76.1-26.el9.x86_64//rpm",
+        "@expat-0__2.5.0-1.el9.x86_64//rpm",
+        "@filesystem-0__3.16-2.el9.x86_64//rpm",
+        "@gawk-0__5.1.0-6.el9.x86_64//rpm",
+        "@gdbm-libs-1__1.19-4.el9.x86_64//rpm",
+        "@glibc-0__2.34-82.el9.x86_64//rpm",
+        "@glibc-common-0__2.34-82.el9.x86_64//rpm",
+        "@glibc-langpack-raj-0__2.34-82.el9.x86_64//rpm",
+        "@gmp-1__6.2.0-13.el9.x86_64//rpm",
+        "@grep-0__3.6-5.el9.x86_64//rpm",
+        "@keyutils-libs-0__1.6.3-1.el9.x86_64//rpm",
+        "@krb5-libs-0__1.21.1-1.el9.x86_64//rpm",
+        "@libacl-0__2.3.1-3.el9.x86_64//rpm",
+        "@libattr-0__2.5.1-3.el9.x86_64//rpm",
+        "@libcap-0__2.48-9.el9.x86_64//rpm",
+        "@libcom_err-0__1.46.5-3.el9.x86_64//rpm",
+        "@libcurl-minimal-0__7.76.1-26.el9.x86_64//rpm",
+        "@libffi-0__3.4.2-8.el9.x86_64//rpm",
+        "@libgcc-0__11.4.1-2.1.el9.x86_64//rpm",
+        "@libnghttp2-0__1.43.0-5.el9.x86_64//rpm",
+        "@libselinux-0__3.5-1.el9.x86_64//rpm",
+        "@libsepol-0__3.5-1.el9.x86_64//rpm",
+        "@libsigsegv-0__2.13-4.el9.x86_64//rpm",
+        "@libtasn1-0__4.16.0-8.el9.x86_64//rpm",
+        "@libuuid-0__2.37.4-15.el9.x86_64//rpm",
+        "@libverto-0__0.3.2-3.el9.x86_64//rpm",
+        "@libxcrypt-0__4.4.18-3.el9.x86_64//rpm",
+        "@mpfr-0__4.1.0-7.el9.x86_64//rpm",
+        "@ncurses-base-0__6.2-10.20210508.el9.x86_64//rpm",
+        "@ncurses-libs-0__6.2-10.20210508.el9.x86_64//rpm",
+        "@openssl-libs-1__3.0.7-24.el9.x86_64//rpm",
+        "@p11-kit-0__0.24.1-2.el9.x86_64//rpm",
+        "@p11-kit-trust-0__0.24.1-2.el9.x86_64//rpm",
+        "@pcre-0__8.44-3.el9.3.x86_64//rpm",
+        "@pcre2-0__10.40-2.el9.x86_64//rpm",
+        "@pcre2-syntax-0__10.40-2.el9.x86_64//rpm",
+        "@python3-0__3.9.17-2.el9.x86_64//rpm",
+        "@python3-libs-0__3.9.17-2.el9.x86_64//rpm",
+        "@python3-pip-wheel-0__21.2.3-7.el9.x86_64//rpm",
+        "@python3-setuptools-wheel-0__53.0.0-12.el9.x86_64//rpm",
+        "@readline-0__8.1-4.el9.x86_64//rpm",
+        "@sed-0__4.8-9.el9.x86_64//rpm",
+        "@setup-0__2.13.7-9.el9.x86_64//rpm",
+        "@sqlite-libs-0__3.34.1-6.el9.x86_64//rpm",
+        "@tzdata-0__2023c-1.el9.x86_64//rpm",
+        "@vim-minimal-2__8.2.2637-20.el9.x86_64//rpm",
+        "@xz-libs-0__5.2.5-8.el9.x86_64//rpm",
+        "@zlib-0__1.2.11-40.el9.x86_64//rpm",
```

Comment:
I'm not opposed to anything added here, although is more than just running python/bash applications.

Reply:
This is the result of doing `make rpm-deps`. I had started off with very minimal stuff I'd need for bash/python. But the make command modified it for me.

Reply:
It should add the dependencies, that's fine. `vim-minimal` is unlikely to be a dependency.

Reply:
This is another place where I spent unreasonable time and then gave up. I agree with your point here. 

So I removed all packages from the `rpms` list, and added only the ones we are interested in - bash, python, centos, and such. Upon doing `make rpm-deps`, the list didn't get populated with additional packages required for dependency. And the resulting container image didn't have the packages required. But I would surely love to see a slimmer image than the one suggested in this PR.

Reply:
Mmh yeah... this isn't very straightforward, I guess we need to create a struct that has a VMI and Configmap object and then you need to append the second object in [`MarshalObject`](https://github.com/kubevirt/kubevirt/blob/main/tools/util/marshaller.go#L32). But I guess this can be left as a TODO for a following PR